packages/helm-charts/load-test/templates/load-test.statefulset.yaml

```diff
@@ -75,47 +78,66 @@ spec:
         - name: data
           mountPath: /root/.celo
 {{ include "common.init-genesis-container" .  | indent 6 }}
-{{ include "common.import-geth-account-container" .  | indent 6 }}
-      containers:
-      - name: geth
+      - name: import-geth-account
         image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         command: ["/bin/sh"]
         args:
         - "-c"
+        - |
+          for thread in $(seq 0 {{ sub .Values.threads 1 | default "0" }}); do
+            geth --nousb account import --password /root/.celo/account/accountSecret /root/.celo/pkey$thread || true
+          done
+          # Josh branch for bulk import
```

Comment:
Should we remove the commented out part?  Or is it intentional to keep for future use?

Reply:
I think we do not need for now. The performance of a account batch importing was considering but not huge and I think we find other problems in other pieces when threads are a high number).